### ISSUE TITLE
hotspot jdk14 build got `Unable to detect git tag`

### ISSUE THREAD

**Issue by sophia-guo** [sophia-guo/build-jdk-original#9]
Build hotspot jdk14 got `Unable to detect git tag,` for all runners

```
Fetching origin
HEAD is now at 3574fa9f64 Merging origin/master HEAD into dev
Pulling latest tags and getting the latest update version using git fetch -q --tags 
NOTE: This can take quite some time!  Please be patient
From /cygdrive/d/a/runaqa/runaqa/openjdk-build/workspace/./build//src
 * branch                  HEAD       -> FETCH_HEAD
WARNING: Failed to identify latest tag in the repository
Unable to detect git tag, exiting... 

```

**Comment by sophia-guo** [comment]
https://github.com/sophia-guo/runaqa/runs/739135626?check_suite_focus=true 

also hit on window.

**Comment by sophia-guo** [comment]
Fixed by #8
